### DISCUSSION TITLE
Arrow keys not displaying on the GUI

### DISCUSSION THREAD

**Discussion by rdhmdhl** [KinesisCorporation/Adv360-Pro-ZMK#573] [Q&A]
When I fork the repo and load it in the GUI, the arrow keys are not displaying. Is there a reason for this?

I tried to create a macro for the arrow keys using the fn button (mapped to layer 1), then I assigned the macros to each arrow key, but it doesn't seem to work.

Here's an image of the arrow keys on layer 0

![Screen Shot 2024-10-10 at 6 56 40 PM](https://github.com/user-attachments/assets/c8239986-a77c-42a3-96a8-6f5a2aad8e67)

Wondering if there's a compatibility issue here or something. I'm on macOS 12.7.6 

Here's a screenshot of layer 1 where I tried to assign the macros to "left" and "left arrow" on separate ocations: 
![Screen Shot 2024-10-10 at 6 53 07 PM](https://github.com/user-attachments/assets/00963a87-6d64-4b87-a489-c984d9c81833)

I've also tried to assign this to the "left" and "left arrow" key options, both without luck. When I'm trying to access the macros, I am holding down the left function key and pressing the left or right arrow...

Any ideas here on what's happening?

**Comment by KinesisCorporation** [comment]
Hi Reid,

This sounds like a browser-specific problem. What browser are you using? They seem to be working fine on Chrome and Safari, but can see a problem on Firefox.
